package.json

```diff
@@ -1,26 +1,44 @@
 {
-  "name": "d2l-localize-behavior",
   "description": "Polymer mixin wrapper around app-localize-behavior which adds automatic language resolution",
-  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BrightspaceUI/localize-behavior.git"
+  },
+  "name": "d2l-localize-behavior",
   "scripts": {
-    "postinstall": "polymer install --variants",
-    "lint:html": "eslint *.html test/*.html",
-    "lint:wc": "polymer lint -i *.html",
-    "lint": "npm run lint:html && npm run lint:wc",
+    "lint:js": "eslint . test/** demo/** --ext .js,.html",
```

Comment:
I think you have an old version of the script, but we don't need the fancy line here (there's no css or scss files in test or demo), so if you don't want to rerun it we can just leave as is and this line will update next time (if we ever have to run it again).